.github/workflows/rust-ci.yml

```diff
@@ -0,0 +1,205 @@
+      - name: Verify prebuilt archive downloaded (GPU, x86_64-pc-windows-msvc)
+        run: ls -lh target/x86_64-pc-windows-msvc/debug/build/onnxruntime-sys-*/out/onnxruntime-win-gpu-x64-1.*.zip
+
+  test:
+    name: Test Suite
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          [
+            x86_64-unknown-linux-gnu,
+            x86_64-apple-darwin,
+            x86_64-pc-windows-msvc,
+            i686-pc-windows-msvc,
+          ]
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+          - target: x86_64-apple-darwin
+            os: macos-latest
+          - target: x86_64-pc-windows-msvc
+            os: windows-latest
+          - target: i686-pc-windows-msvc
+            os: windows-latest
+    env:
+      CARGO_BUILD_TARGET: ${{ matrix.target }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: ${{ matrix.target }}
+          override: true
+      - name: Install additional packages (macOS)
+        if: contains(matrix.target, 'x86_64-apple-darwin')
+        run: brew install libomp
+      - name: Build (cargo build)
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --all --manifest-path ${{ env.MANIFEST_PATH }}
+      - name: Build tests (cargo test)
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --no-run --manifest-path ${{ env.MANIFEST_PATH }}
+      - name: Build onnxruntime with 'model-fetching' feature
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          # Use --manifest-path instead of --package. See https://github.com/actions-rs/cargo/issues/86
+          args: --manifest-path ${{ env.MANIFEST_PATH }} --features model-fetching
+      - name: Test onnxruntime-sys
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --package onnxruntime-sys -- --test-threads=1 --nocapture
+      - name: Test onnxruntime
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          # Use --manifest-path instead of --package. See https://github.com/actions-rs/cargo/issues/86
+          args: --manifest-path $MANIFEST_PATHl --features model-fetching -- --test-threads=1 --nocapture
+
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - run: rustup component add clippy
+      - uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-features --manifest-path ${{ env.MANIFEST_PATH }} -- -D warnings
+
+  coverage:
```

Comment:
Probably should remove this section initially and determine if this is the path for future code coverage metrics.